core/src/services/azblob/backend.rs

```diff
@@ -644,6 +645,7 @@ impl Access for AzblobBackend {
                 self.core
                     .azblob_put_blob_request(path, None, &OpWrite::default(), Buffer::new())?
             }
+            PresignOperation::Delete(_) => todo!(),
```

Comment:
We should avoid returning `todo!()` here, as it can cause the user's application to panic. Instead, return an `Unsupported` error.

This comment applies to all services.